boards/arm/sam4s_xplained/sam4s_xplained_defconfig

```diff
@@ -0,0 +1,8 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_SAM=y
+CONFIG_SOC_SERIES_SAM4S=y
+CONFIG_SOC_PART_NUMBER_SAM4S16C=y
+CONFIG_BOARD_SAM4S_XPLAINED=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CONSOLE=y
+CONFIG_PINMUX=y
```

Comment:
That's probably not the Zephyr policy but diff works much nicer if compared files have newline at the end of file.